lib/boreale/users.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Boreale.Users do
```

Comment:
Moved user/domain logic to their own module for better organization but mainly because `mock` redefines module as mock. So mocking LoginController's function redefines a new empty module with only mocked functions.